index.html

```diff
@@ -60,7 +60,7 @@
           <h1>Hey There,<br>
             I'm Abdus Sattar
           </h1>
-          <h3>I'm a Software Developer</h3>
+          <h2>I'm a Software Developer</h2>
```

Comment:
- [ Required ] It would be better if this text is made more visible, currently, It does not follow the `Figma template` design which has a `#fff` color. Please see the screenshot below for more details on this and rectify it.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/215787119-d95012f8-cf86-4893-bbf5-d2300f1cb504.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/215786086-82d07f4c-8723-4a69-aa9a-8345bbe45e61.png" width="375px" />